### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Kate-Lyndegaard**
In this PR, I reviewed/fixed all layer titles and rule titles.

### CODE REVIEW

File: feature-styles/HH_HealthDeterminantMeasure.se [hunk 1]
```diff
@@ -14,7 +14,7 @@ version="1.1.0" xsi:schemaLocation="http://www.opengis.net/se http://schemas.ope
   <se:FeatureTypeName>hh:EnvHealthDeterminantMeasure</se:FeatureTypeName>
   <se:Rule>
     <se:Description>
-      <se:Title>human health: polygon</se:Title>
+      <se:Title>Human health polygon</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HH_HealthDeterminantMeasure.se:17]: For Renewable and Waste Resource, polygonS and lineS are used in plural - here they are used in singular - maybe align?

File: feature-styles/PS_ProtectedSites.se [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <se:FeatureTypeName>ps:ProtectedSite</se:FeatureTypeName>
   <se:Rule>
     <se:Description>
-      <se:Title>protected sites: polygon</se:Title>
+      <se:Title>Protected sites: polygon</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/PS_ProtectedSites.se:11]: For other FeatureTypes with different geometries the colon between Type and geometry has been removed - maybe align.

File: feature-styles/HY_P_WaterbodiesPersistence.se [hunk 3]
```diff
@@ -48,7 +48,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
     <se:Description>
-      <se:Title>standing waters persistence: perennial</se:Title>
+      <se:Title>Standing waters persistence: perennial</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HY_P_WaterbodiesPersistence.se:51]: For other cases with different geometries, the geometry was mentioned in the title - maybe align?

File: feature-styles/HY_P_WatercoursePersistence.se [hunk 4]
```diff
@@ -44,7 +44,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
      <se:Description>
-      <se:Title>waterbodies persistence: perennial</se:Title>
+      <se:Title>Waterbodies persistence: perennial</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HY_P_WatercoursePersistence.se:47]: For other cases with different geometries, the geometry was mentioned in the title - maybe align?

File: feature-styles/HY_P_WatercoursePersistence.se [hunk 5]
```diff
@@ -108,7 +108,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
       <se:Description>
-      <se:Title>waterbodies persistence: intermittent</se:Title>
+      <se:Title>Waterbodies persistence: intermittent</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HY_P_WatercoursePersistence.se:111]: For other cases with different geometries, the geometry was mentioned in the title - maybe align?

File: feature-styles/HY_P_WatercoursePersistence.se [hunk 6]
```diff
@@ -180,7 +180,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
     <se:Description>
-      <se:Title>waterbodies persistence: ephemeral or dry</se:Title>
+      <se:Title>Waterbodies persistence: ephemeral or dry</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HY_P_WatercoursePersistence.se:183]: For other cases with different geometries, the geometry was mentioned in the title - maybe align?

File: feature-styles/HY_P_Watercourse_ManMade.se [hunk 7]
```diff
@@ -44,7 +44,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
      <se:Description>
-      <se:Title>waterbodies natural</se:Title>
+      <se:Title>Waterbodies natural</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HY_P_Watercourse_ManMade.se:47]: For other cases with different geometries, the geometry was mentioned in the title - maybe align?

File: feature-styles/HY_P_Watercourse_ManMade.se [hunk 8]
```diff
@@ -107,7 +107,7 @@ version="1.1.0">
     </se:Rule>
     <se:Rule>
         <se:Description>
-      <se:Title>waterbodies man-made</se:Title>
+      <se:Title>Waterbodies man-made</se:Title>
```
Comment (JohannaOtt) [inline feature-styles/HY_P_Watercourse_ManMade.se:110]: For other cases with different geometries, the geometry was mentioned in the title - maybe align?

**Review by JohannaOtt — CHANGES_REQUESTED** [review]
I found some things, where the different titles are not aligned but I don't know if that is maybe a result of different rules in the DS.
I did not do a detailed review as that would include publishing datasets for all themes and I guess we can still react here in case issues occur in the future. What do you think  @Kate-Lyndegaard ?

**Comment by Kate-Lyndegaard** [comment]
> 
> 
> I found some things, where the different titles are not aligned but I don't know if that is maybe a result of different rules in the DS.
> I did not do a detailed review as that would include publishing datasets for all themes and I guess we can still react here in case issues occur in the future. What do you think @Kate-Lyndegaard ?

I think this is great and I knew you would comment on this:-O
I am also unsure how to handle this. In the original template files, some INSPIRE SLDs referenced the geometry type in the rule title. This means that it will appear in the legend as follows:

<square icon> protected sites: polygon
<line icon> protected sites: line.... etc

The textual description of the geometry seems redundant because you already have the icon clearly indicating the geometry type!
If you remove the geometry type, however, then you have three identical labels: 

<square icon> protected sites
<line icon> protected sites

**Comment by JohannaOtt** [comment]
I personally would prefer to add the geometries to the legend - I think identical labels look a bit weird. So even if it is obvious with the symbols, I would add the geometry type.

**Review by JohannaOtt — APPROVED** [review]

**Pull request merged**